README.md

```diff
@@ -26,3 +26,51 @@ const posAutCode = "123456789ssA";
 const sisp = new Sisp({ posID, posAutCode });
 
 ```
+
```

Comment:
The README has a few of issues:
1. `const sisp` instead of `const Sisp` again.
2. We don’t need function documentation in the README - just examples and/or descriptions.
3. How to configure the project is repeated.

Reply:
@nnascim `const sisp` is lower case because it is an instance of a class